tests/phpunit/tests/query/lazyLoadTermMeta.php

```diff
@@ -133,6 +162,6 @@ public function test_wp_queue_posts_for_term_meta_lazyload_delete_term() {
 		$args     = $filter->get_args();
 		$first    = reset( $args );
 		$term_ids = end( $first );
-		$this->assertNotContains( $remove_term_id, $term_ids );
+		$this->assertContains( $remove_term_id, $term_ids );
```

Comment:
This change seems incorrect.

`$remove_term_id` has been deleted on L150 so the lazy loader shouldn't attempt to include it.

Reply:
@peterwilsoncc Yeah you are right, this is weird isn't it. But there is no code the lazy load meta api, to remove deleted terms / term meta from the lazy load queue. 

For context, this tests was somewhat broken in trunk. It only checks to the first element in the array. There were two in that array. Now that there are only one, a positive change, the removed id is present.

Reply:
Thanks, I tested against trunk and the feature branch by adding the following to the test:

```php
		$pwcc_filter = function( $null, $object_ids ) use ( $remove_term_id ) {
			var_dump( [ 'object_ids' => $object_ids, 'removed' => $remove_term_id, 'in' => in_array( $remove_term_id, $object_ids, true ) ] );
			return $null;
		};

		add_filter( 'update_term_metadata_cache', $pwcc_filter, 10, 2 );
```

[The output shows](https://gist.github.com/peterwilsoncc/90fc5e6560cc01e0406e6e97333bb005) that on both branches, two calls are made to warm the meta data:
* on trunk: the deleted object ID is only included in the second call (currently untested)
* on the feature branch: the deleted object ID is included in both the first and second call

So yeah, flakey test.